### ISSUE TITLE
Mathjax

### ISSUE THREAD

**Issue by ps-george** [ymherklotz/FMark#61]

**Comment by ymherklotz** [comment]
https://docs.mathjax.org/en/latest/start.html

```html
<!DOCTYPE html>
<html>
<head>
<title>MathJax TeX Test Page</title>
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}});
</script>
<script type="text/javascript" async
  src="https://example.com/MathJax.js?config=TeX-AMS_CHTML">
</script>
</head>
<body>
When $a \ne 0$, there are two solutions to \(ax^2 + bx + c = 0\) and they are
$$x = {-b \pm \sqrt{b^2-4ac} \over 2a}.$$
</body>
</html>
```

**Comment by ps-george** [comment]
Implemented in #134
